pkg/provider/parse.go

```diff
@@ -194,6 +194,8 @@ func validateProviderType(config *ProviderConfig) error {
 		if len(config.Agent.Custom.CommandDevContainer) == 0 {
 			return fmt.Errorf("agent.custom.commandDevContainer is required")
 		}
+		// TODO: Add config.Agent.Custom.GetDevContainerLogs validation
```

Comment:
will open corresponding kubernetes provider PR once this is merged, then release kubernetes provider and _then_ enable validation after 2 weeks or so